UM/Application.py

```diff
@@ -44,13 +45,15 @@ class Application:
     #   \param version \type{string} Version, formatted as major.minor.rev
     #   \param build_type Additional version info on the type of build this is, such as "master".
     #   \param is_debug_mode Whether to run in debug mode.
-    def __init__(self, name: str, version: str, app_display_name: str = "", build_type: str = "", is_debug_mode: bool = False, **kwargs) -> None:
+    def __init__(self, name: str, version: str, api_version: str = "5.0.0", app_display_name: str = "", build_type: str = "", is_debug_mode: bool = False, **kwargs) -> None:
```

Comment:
I think passing a default value here is dangerous as it can unexpectedly set the SDK version without knowing about it.

Reply:
I agree. Fortunately, nothing else seems to depend on this "hard-coded" value. Changed this in 78b8372c